### ISSUE TITLE
Selenium 3.x V is keep skipping the test execution after Firefox 53v launch

### ISSUE THREAD

**Issue by java123code** [SeleniumHQ/selenium#4200]
1. below is the error message I can see 

2. My Config :
JAVA - JRE (Java Runtime Environment) 8, JDK (Java Development Kit) jdk1.8.0_60
TestNG Framework
Selenium 3.3 or 3.4 ( if not then 3 or above )
Mozilla Firefox 53.0 or 54.0
Apache Ant 1.9.7
Appium version 1.4.13
gecko driver : V 14, 15 , 16 or 17 
log4j : 1.2.17

3. Issue : I can launch the browser as well as I can login the site but after few activities my test is keep
skipping the further execution steps. The only thing I can highlight is I needed to add Selenium 2.45 too in order to remove some of errors in my java modules. 

1497657743317 geckodriver   INFO   Listening on 127.0.0.1:30220
1497657743880 geckodriver::marionette    INFO   Starting browser \\?\C:\Program Files (x86)\Mozilla Firefox\firefox.exe with args ["-marionette"]
1497657744925 addons.manager       ERROR  startup failed: [Exception... "Component returned failure code: 0x80070057 (NS_ERROR_ILLEGAL_VALUE) [nsIFile.create]"  nsresult: "0x80070057 (NS_ERROR_ILLEGAL_VALUE)"  
location: "JS frame :: resource://gre/modules/FileUtils.jsm :: FileUtils_getDir :: line 70"  data: no] Stack trace: FileUtils_getDir()@resource://gre/modules/FileUtils.jsm:70 < 
FileUtils_getFile()@resource://gre/modules/FileUtils.jsm:42 < validateBlocklist()@resource://gre/modules/AddonManager.jsm:674 < startup()@resource://gre/modules/AddonManager.jsm:837 < startup()@resource://gre/modules/AddonManager.jsm:3145 
< observe()@resource://gre/components/addonManager.js:65
JavaScript error: resource://gre/modules/AddonManager.jsm, line 1657: NS_ERROR_NOT_INITIALIZED: AddonManager is not initialized
1497657747357 Marionette    INFO   Listening on port 51995
JavaScript error: resource://gre/modules/AddonManager.jsm, line 2572: NS_ERROR_NOT_INITIALIZED: AddonManager is not initialized
Jun 16, 2017 5:02:27 PM org.openqa.selenium.remote.ProtocolHandshake createSession
INFO: Detected dialect: W3C

                     setUp | Firefox browser launched
            ManageBrowser | Implicit wait applied on the driver for 60 seconds
          ManageBrowser | Browser window is maximized
JavaScript error: https://xx.xxxx.com/includes/js/s_code.js, line 1: SyntaxError: unterminated comment
|06-16-2017 17:02:29 |  INFO |                          ManageBrowser | Web application launched successfully

**Comment by java123code** [comment]
I have the test cases. I have all set up and version jars mentioned above for selenium , appium etc. Issue I am reporting is I can not execute the test case completely. The execution is stopped once my script logged in the site and right after that I can see skip status and test is keep trying to execute remaining steps but can not see correct activity on webpage. So copied the error message I have found in logs. Please advise. thanks

**Comment by p0deje** [comment]
Please provide [concise reproducible test case](http://sscce.org/) so that we could act on this issue.

See [CONTRIBUTING.md](https://github.com/SeleniumHQ/selenium/blob/master/CONTRIBUTING.md#issue-contributions)

**Comment by java123code** [comment]
if you have all of above mentioned configuration in your automation then : 
1. launch the firefox browser 
2. Do some activity on web page
3. collect logs in the back ground 

since its a GUI tool so too much code involved and lots of jars imported that's where it challenging to troubleshoot.

**Comment by AutomatedTester** [comment]
Unfortunately every site is unique as is every configuration. If you will find it troublesome to narrow down I guarantee we will too

**Comment by java123code** [comment]
No, I think I made very confusing comments above, sorry for that. The configuration I have provided in the ticket is for my Automation tool (not for any website ), this tool is really complex and for multi purpose - for example its GUI, good for web site / mobile devices , collecting logs etc. 

so My tool is good below 3.x Selenium for firefox 46 below but I can not run this tool successfully with 3.x Above and firefox 53 above. 

I hope now its clear, there must be compatibility issue. I am pretty sure it's not code related since tool is able to launch the browser successfully and once it's logged in with the account info it does nothing after that, just stays on webpage.
